src/components/HappySlider/HappySlider.scss

```diff
@@ -0,0 +1,58 @@
+:root {
+  /*Variables setup for switching images on slider thumb*/
+  --background-image-clean: url('./Smileys/smiley1.svg');
+  --background-image-noise: url('./Smileys/smiley2.svg');
+  --background-image-seating: url('./Smileys/smiley3.svg');
+  --background-image-wifi: url('./Smileys/smiley5.svg');
+}
+
+input[type=range] {
+  -webkit-appearance: none;  /* Override default CSS styles */
+  appearance: none;
+  width: 100%; /* Full-width */
+  height: 8px; /* Specified height */
+  background: #0000FF; /* #0000FF background */
+  border-radius: 5px;
+  outline: none; /* Remove outline */
+  opacity: 0.7; /* Set transparency (for mouse-over effects on hover) */
+  -webkit-transition: .2s; /* 0.2 seconds transition on hover */
+  transition: opacity .2s;
+}
+
+input[type=range]:hover {
+  opacity: 1; /* Fully shown on mouse-over */
+}
+
+input[type=range]::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 24px; /* width of thumb */
+  height: 24px; /* height of thumb */
+  border: 3px solid #0000FF;
+  border-radius: 15px; /* Create circle for thumb image */
+  background: #0000FF; /* Back-up coloring if image too small */
+  background-size: 18px; /* Size of image inside thumb */
```

Comment:
`background-size` handles the image for the slider thumb. With the thumb width and height being 24px X 24 px with a border of 3px(6px subtracted from the circle), that leaves 18px left over for the image. Importing images that are perfectly centered will render a nicely aligned image, but if an image is imported without being perfectly centered, is there a way to remedy that?

Reply:
Could you slack me a photo of different behaviors for these images? I'm a little confused about how they're behaving.

Reply:
If you look at the photo in my first comment, you can see that the images are slightly favoring the bottom of the thumb. The images are also a touch small. By changing the background-color to match the slider track, I was able to hide that discrepancy. If I change the background color you can see what I mean. 

<img width="115" alt="Screen Shot 2020-07-21 at 3 45 58 PM" src="https://user-images.githubusercontent.com/54245791/88099635-6e414f80-cb69-11ea-8b18-fc3a7ce2007f.png">